rust/src/measures/ffi.rs

```diff
@@ -314,7 +314,42 @@ pub extern "C" fn opendp_measures__user_divergence(
     descriptor: *mut c_char,
 ) -> FfiResult<*mut AnyMeasure> {
     let descriptor = try_!(to_str(descriptor)).to_string();
-    Ok(AnyMeasure::new(UserDivergence { descriptor })).into()
+    Ok(AnyMeasure::new(ExtrinsicDivergence { descriptor })).into()
+}
+
+#[bootstrap(
+    name = "_extrinsic_divergence",
+    arguments(descriptor(rust_type = "String"))
+)]
+/// Construct a new ExtrinsicDivergence, a privacy measure defined from a bindings language.
+/// This is meant for internal use, as it does not require "honest-but-curious",
+/// unlike `user_divergence`.
+///
+/// Any two instances of ExtrinsicDivergence are equal if their string descriptors are equal.
+///
+/// The essential requirement of a privacy measure is that it is closed under postprocessing.
```

Comment:
Here and elsewhere would it be good to just reference the corresponding docs for the `user` function to avoid the copy and paste?